deploy/moac-deployment.yaml

```diff
@@ -23,7 +23,6 @@ spec:
             - "--v=2"
             - "--csi-address=$(ADDRESS)"
             - "--feature-gates=Topology=true"
-            - "--strict-topology=false"
```

Comment:
Hey Tiago, my fault. This should actually stay here and it should be added to our chart template. I have this change in my PR so if it gets merged before yours then you can simply omit this change.